internal/docs/readme.go

```diff
@@ -160,7 +170,7 @@ func findReadmeTemplatePath(fileName, packageRoot string) (string, bool, error)
 	return templatePath, true, nil
 }
 
-func renderReadme(fileName, packageRoot, templatePath string) ([]byte, error) {
+func renderReadme(fileName, packageRoot, templatePath string, linksMap linkMap) ([]byte, error) {
```

Comment:
Do you think we can stop passing the `linksMap` through all these functions and just define it in the deepest function?

Reply:
I wanted to try to do that, but doing so the links file would need to be read again for each markdown file. That's why I read that file at the beginning of the process to be used for all of them, since those links are not going to change.

Currently, there are not so many readme files per package, and maybe it would be preferable to move for now this read to the deepest function. WDYT?

Reply:
Updated. I've just kept linksMap as a parameter in renderReadme function, so testing could be done easier.